commit ad416c8656c2dff6f1bcffee8517284de94c9d92
Author: zoulasc <christos@zoulas.com>
Date:   Thu May 12 10:21:55 2016 -0400

    Update the ctime of the directory too, since it changed. From Mark Davies

diff --git a/amd/autil.c b/amd/autil.c
index 35d8708..f44a0e2 100644
--- a/amd/autil.c
+++ b/amd/autil.c
@@ -183,6 +183,7 @@ forcibly_timeout_mp(am_node *mp)
      * more.
      */
     clocktime(&mp->am_parent->am_fattr.na_mtime);
+    mp->am_parent->am_fattr.na_ctime = mp->am_parent->am_fattr.na_mtime;
     reschedule_timeout_mp();
   }
 }
@@ -312,8 +313,10 @@ am_mounted(am_node *mp)
   /*
    * Update mtime of parent node (copying "struct nfstime" in '=' below)
    */
-  if (mp->am_parent && mp->am_parent->am_al->al_mnt)
+  if (mp->am_parent && mp->am_parent->am_al->al_mnt) {
     mp->am_parent->am_fattr.na_mtime = mp->am_fattr.na_mtime;
+    mp->am_parent->am_fattr.na_ctime = mp->am_fattr.na_mtime;
+  }
 
   /*
    * This is ugly, but essentially unavoidable
